carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -84,7 +84,7 @@ services:
       - /opt/carma/vehicle:/opt/carma/vehicle
     command: bash -c 'source /opt/carma/install/setup.bash && ros2 launch /opt/carma/vehicle/config/drivers.launch.py drivers:=dsrc_driver'
 
-  torc-pinpoint-driver:
+  torc-pinpoint_driver:
```

Comment:
Just confirming whether this is a functional change, or whether it's just to maintain consistency with the other service names?

Additionally, just a small typo: first hyphen needs to be changed to an underscore